dev-libs/aws-sdk-cpp/aws-sdk-cpp-1.5.17.ebuild

```diff
@@ -13,26 +13,32 @@ LICENSE="Apache-2.0"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 
-ALL_AWS_TARGETS="access-management acm acm-pca alexaforbusiness apigateway application-autoscaling appstream appsync athena autoscaling autoscaling-plans AWSMigrationHub batch budgets ce cloud9 clouddirectory cloudformation cloudfront cloudhsm cloudhsmv2 cloudsearch cloudsearchdomain cloudtrail codebuild codecommit codedeploy codepipeline codestar cognito-identity cognito-idp cognito-sync comprehend config connect core cur datapipeline dax devicefarm directconnect discovery dms ds dynamodb dynamodbstreams ec2 ecr ecs eks elasticache elasticbeanstalk elasticfilesystem elasticloadbalancing elasticloadbalancingv2 elasticmapreduce elastictranscoder email es events firehose fms gamelift glacier glue greengrass guardduty health iam identity-management importexport inspector iot iot1click-devices iot1click-projects iotanalytics iot-data iot-jobs-data kinesis kinesisanalytics kinesisvideo kinesis-video-archived-media kinesis-video-media kms lambda lex lex-models lightsail logs machinelearning macie marketplacecommerceanalytics marketplace-entitlement mediaconvert medialive mediapackage mediastore mediastore-data mediatailor meteringmarketplace mobile mobileanalytics monitoring mq mturk-requester neptune opsworks opsworkscm organizations pi pinpoint polly pricing queues rds redshift rekognition resource-groups resourcegroupstaggingapi route53 route53domains s3 s3-encryption sagemaker sagemaker-runtime sdb secretsmanager serverlessrepo servicecatalog servicediscovery shield sms snowball sns sqs ssm states storagegateway sts support swf text-to-speech transcribe transfer translate waf waf-regional workdocs workmail workspaces xray"
```

Comment:
Firstly, please don't change the old version in place. Instead, let's reduce this commit to updating the newly committed version.

Reply:
I'd to drop the older revision since I wanted to avoid two entries for the same flag in metadata.xml

In particular, I had tried a TARGETS approach previously but that fails since AWS_TARGETS is not a current USE_EXPAND flag. So the use flags were renamed from aws_target_module to aws_module.

Reply:
You don't need any prefix, tbh. So just make it 'access-management' and so on, it will be shorter.

Reply:
I have now :)